src/composable_kernel/external/rocm/include/bfloat16_dev.hpp

```diff
@@ -30,7 +30,7 @@
 extern "C" {
 #endif
 
-#ifdef __HIP_PLATFORM_HCC__
+#ifdef __HIP_PLATFORM_AMD__
```

Comment:
We must check offline compilation with ROCm 5.7. If 5.7 offline compiler does not define `__HIP_PLATFORM_AMD__`, then we'll have backward compatibility problem.

Reply:
According to the documentation, this should work down to ROCm 5.0.0 at least (as it defines __HIP_PLATFORM_AMD__).

Reply:
Yes I checked with compiler folks before making these changes :)

Reply:
[resolved]